dietpi/dietpi-software

```diff
@@ -12465,6 +12484,47 @@ _EOF_
 
 		fi
 
+    software_id=181 # Paper
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+
+			# Make sure user agrees to the EULA
+			G_WHIP_BUTTON_OK_TEXT='YES'
+			G_WHIP_BUTTON_CANCEL_TEXT='NO'
+			if G_WHIP_YESNO "Do you agree to the Minecraft EULA found at.
+\nhttps://account.mojang.com/documents/minecraft_eula"; then
+        java -jar /usr/local/bin/paper/paperclip.jar nogui
+				echo eula=true > /usr/local/bin/paper/eula.txt
+        if G_WHIP_YESNO "Would you like to install the Geyser and Floodgate
+        \n plugins for compatibility with Bedrock Edition?"; then
+            curl https://ci.nukkitx.com/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar -o /usr/local/bin/paper/plugins/Geyser-Spigot.jar
+            curl https://ci.nukkitx.com/job/GeyserMC/job/Floodgate/job/development/lastSuccessfulBuild/artifact/bukkit/target/floodgate-bukkit.jar -o /usr/local/bin/paper/plugins/floodgate-bukkit.jar
+        fi
+        # Service
+			cat << '_EOF_' > /etc/systemd/system/paper.service
+[Unit]
+Description=paper (DietPi)
+
+[Service]
+SyslogIdentifier=papermc
+WorkingDirectory=/usr/local/bin/paper
+ExecStart=/bin/bash -c 'java -jar /usr/local/bin/paper/paperclip.jar'
```

Comment:
Any reason to wrap the process into another ~4 MiB bash shell? I'd just go with:
```
ExecStart=$(command -v java) -jar /path/to/paperclip.jar'
```
`'_EOF_'` => `_EOF_` required then to expand the Java executable path.